R/define.R

```diff
@@ -74,20 +78,25 @@ define_enroll_rate <- function(
   df
 }
 
-#' Define fail rate
+#' Define failure rate
+#'
+#' The function define subject failure rate for a study with two treatment groups.
+#' It also supports stratified design that have different failure rate in
+#' each stratum.
+#'
 #'
 #' @param duration A numeric vector of piecewise study duration interval.
-#' @param fail_rate A numeric vector of failure rate in each `duration`.
+#' @param fail_rate A numeric vector of failure rate in each `duration` in the control group.
 #' @param dropout_rate A numeric vector of dropout rate in each `duration`.
-#' @param hr A numeric vector of hazard ratio.
+#' @param hr A numeric vector of hazard ratio between case and control group.
```

Comment:
gsDesign and gsDesign2 both used "treatment group" instead of "case group" consistently. There was not a single "case group" usage.